src/main/java/com/jcabi/github/Status.java

```diff
@@ -52,57 +53,82 @@
      * Get URL.
      * @return URL as string.
      */
-    String targetUrl();
+    Optional<String> targetUrl();
 
     /**
      * Get description.
      * @return Description as string.
      */
-    String description();
+    Optional<String> description();
 
     /**
      * Get context.
      * @return Context as string
      */
-    String context();
+    Optional<String> context();
 
     /**
      * States of Status API.
      * @author Marcin Cylke(marcin.cylke+github@gmail.com)
      * @version $Id$
      */
-    public enum State {
+    enum State implements StringEnum {
         /**
          * Pending state.
          */
-        Pending,
+        Pending("pending"),
         /**
          * Success state.
          */
-        Success,
+        Success("success"),
         /**
          * Error state.
          */
-        Error,
+        Error("error"),
         /**
          * Failure state.
          */
-        Failure;
+        Failure("failure");
```

Comment:
@cvrebert as far as I know is a convention to make all enum items in uppercase...